db/migrate/20210911182500_add_columns_for_case_development_analytics.rb

```diff
@@ -0,0 +1,18 @@
+class AddColumnsForCaseDevelopmentAnalytics < ActiveRecord::Migration[6.1]
+  def change
```

Comment:
before I forget, we need to disable active record timestamps before and after these changes:

```
    ActiveRecord::Base.record_timestamps = false
```

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1085/commits/e34c127e1ddb83a1cced5e61c4ba0b3bbf9b3ccd